.github/workflows/int-and-connected-test-run.yml

```diff
@@ -145,44 +194,71 @@ jobs:
           echo "::set-output name=git_hash::$git_hash"
           echo "Latest git hash for this branch: $git_hash"
       - name: "Wait for deployment to come back online"
-        uses: broadinstitute/datarepo-actions/actions/wait-for-deployment@0.62.0
+        uses: broadinstitute/datarepo-actions/actions/wait-for-deployment@dr-2757-nm-better-unit
         timeout-minutes: 20
         env:
           DESIRED_GITHASH: ${{ steps.configuration.outputs.git_hash }}
           DEPLOYMENT_TYPE: 'api'
       - name: "Run Test Runner smoke tests via Gradle"
-        uses: broadinstitute/datarepo-actions/actions/main@0.62.0
+        uses: broadinstitute/datarepo-actions/actions/main@dr-2757-nm-better-unit
         with:
           actions_subcommand: 'gradletestrunnersmoketest'
-      - name: "Run Integration test via Gradle"
-        uses: broadinstitute/datarepo-actions/actions/main@0.62.0
+      - name: "Run integration tests via Gradle"
+        uses: broadinstitute/datarepo-actions/actions/main@dr-2757-nm-better-unit
         with:
           actions_subcommand: 'gradleinttest'
           pgport: ${{ job.services.postgres.ports[5432] }}
           test_to_run: 'testIntegration'
+      - name: "Clean state lock from used Namespace on API deploy"
+        if: always()
+        uses: broadinstitute/datarepo-actions/actions/main@dr-2757-nm-better-unit
+        with:
+          actions_subcommand: 'k8_checknamespace_clean'
+      - name: "Clean whitelisted Runner IP"
+        if: always()
+        uses: broadinstitute/datarepo-actions/actions/main@dr-2757-nm-better-unit
+        with:
+          actions_subcommand: 'gcp_whitelist_clean'
```

Comment:
Instead of doing this, we made our environments public. Not sure if this is possible for TDR, but it dramatically simplifies the actions because they don't need to whitelist or clean up.

Reply:
Yeah, I talked to Mike from devops about this today.  I think we could do something to the cluster with a different identity system